tests/scripts/thread-cert/base_v1_2_LowPower_6_1_07_PreferringARouterOverAReed.py

```diff
@@ -0,0 +1,160 @@
+        self.nodes[DUT].start()
+        self.simulator.go(5)
+        self.assertEqual(self.nodes[DUT].get_state(), 'child')
+
+    def verify(self, pv):
+        print("Run verify")
+        pkts = pv.pkts
+        pv.summary.show()
+        LEADER = pv.vars['LEADER']
+        ROUTER_1 = pv.vars['ROUTER_1']
+        REED_1 = pv.vars['REED_1']
+        DUT = pv.vars['DUT']
+
+        # Step 1 - Ensure all routers and leader are sending MLE advertisements
+        pkts.filter_wpan_src64(LEADER) \
+            .filter_mle_cmd(consts.MLE_ADVERTISEMENT) \
+            .must_next()
+        pkts.filter_wpan_src64(ROUTER_1) \
+            .filter_mle_cmd(consts.MLE_ADVERTISEMENT) \
+            .must_next()
+
+        # Step 3 - DUT begins attach process by sending a multicast MLE Parent Request
+        pkts.filter_wpan_src64(DUT) \
+            .filter_mle_cmd(consts.MLE_PARENT_REQUEST) \
+            .filter_mle_has_tlv(TlvType.CHALLENGE) \
+            .filter_mle_has_tlv(TlvType.MODE) \
+            .filter(lambda p: p.mle.tlv.scan_mask.r == 1 and p.mle.tlv.scan_mask.e == 0) \
+            .filter(lambda p: p.mle.tlv.version == 3) \
+            .filter(lambda p: p.ipv6.hlim == 255) \
+            .filter_LLARMA() \
+            .must_next()
+
+        # Step 4 - Router_1 automatically Respond with MLE Parent Response
+        pkts.filter_wpan_src64(ROUTER_1) \
+            .filter_mle_cmd(consts.MLE_PARENT_RESPONSE) \
+            .filter(lambda p: p.mle.tlv.version == 2) \
+            .must_next()
+
+        # Step 5 - DUT sends another multicast MLE Parent Request to the all-routers multicast with the Scan Mask TLV
+        # set for all routers and all REEDs
+        pkts.filter_wpan_src64(DUT) \
+            .filter_mle_cmd(consts.MLE_PARENT_REQUEST) \
+            .filter_mle_has_tlv(TlvType.CHALLENGE) \
+            .filter_mle_has_tlv(TlvType.MODE) \
+            .filter(lambda p: p.mle.tlv.scan_mask.r == 1 and p.mle.tlv.scan_mask.e == 1) \
+            .filter(lambda p: p.mle.tlv.version == 3) \
+            .filter(lambda p: p.ipv6.hlim == 255) \
+            .filter_LLARMA() \
+            .must_next()
+
+        # Step 6 - REED_1 sends MLE Parent Response to the DUT
+        pkts.filter_wpan_src64(REED_1) \
+            .filter_mle_cmd(consts.MLE_PARENT_RESPONSE) \
+            .must_next()
+
+        # # Step 7 - DUT sends a Child ID Request to REED_1 due to its better link quality
+        pkts.filter_wpan_src64(DUT) \
+            .filter_mle_cmd(consts.MLE_CHILD_ID_REQUEST) \
+            .filter_wpan_dst64(ROUTER_1) \
+            .filter_mle_has_tlv(TlvType.ADDRESS_REGISTRATION) \
+            .filter_mle_has_tlv(TlvType.LINK_LAYER_FRAME_COUNTER) \
+            .filter_mle_has_tlv(TlvType.MODE) \
+            .filter_mle_has_tlv(TlvType.RESPONSE) \
+            .filter_mle_has_tlv(TlvType.TIMEOUT) \
+            .filter_mle_has_tlv(TlvType.TLV_REQUEST) \
```

Comment:
```suggestion
            .filter_mle_has_tlv(TlvType.ADDRESS_REGISTRATION, TlvType.LINK_LAYER_FRAME_COUNTER, TlvType.MODE, TlvType.RESPONSE, TlvType.TIMEOUT, TlvType.TLV_REQUEST) \
```

allow `filter_mle_has_tlv` to accept one or more TLV types?